src/components/ui/ListEntry.js

```diff
@@ -46,13 +46,14 @@ const TextContainer = styled.div`
   }
 `
 
+// Forwarding ref here seems necessary for Combobox highlighting to work
 const ListEntry = React.forwardRef(
   ({ leftIcons, primaryText, secondaryText, rightIcons, ...props }, ref) => (
     <ListContainer ref={ref} {...props}>
       <Icons>{leftIcons}</Icons>
       <TextContainer>
-        <PrimaryText>{primaryText}</PrimaryText>
-        <SecondaryText>{secondaryText}</SecondaryText>
+        <PrimaryText className="primaryText">{primaryText}</PrimaryText>
```

Comment:
I don't think there's anything wrong with this, but I was wondering how having the className changes this.

Reply:
@vasuchal

Reply:
Good question!!

Reply:
We need that so that we can override styles on ListEntry in the AccordionButton using CSS selectors.